config/ignition_citadel_gazebo11_debian_buster.yaml

```
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/debian-stable
+(Package (% libignition-plugin, $Version (% 1.1.0*))|\
+(Package (% libignition-plugin-dev, $Version (% 1.1.0*))|\
+(Package (% libignition-rendering3, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-core-dev, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-dev, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-ogre1, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-ogre1-dev, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-ogre2, $Version (% 3.2.0*))|\
+(Package (% libignition-rendering3-ogre2-dev, $Version (% 3.2.0*))|\
+(Package (% libignition-sensors3, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-air-pressure, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-air-pressure-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-altimeter, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-altimeter-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-core-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-depth-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-depth-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-gpu-lidar, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-gpu-lidar-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-imu, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-imu-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-lidar, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-lidar-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-logical-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-logical-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-magnetometer, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-magnetometer-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-rendering, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-rendering-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-rgbd-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-rgbd-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-thermal-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-thermal-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-tools-dev, $Version (% 1.0.0*))|\
+(Package (% libignition-transport8, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-core-dev, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dbg, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dev, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log-dev, $Version (% 8.0.0*))|\
+(Package (% libogre-2.1)|\
```

Not sure if we want versions for ogre too, I can not think a reason not to do it:
```suggestion
(Package (% libogre-2.1, $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```